src/views/RequestView.py

```diff
@@ -25,6 +25,12 @@ def create():
 
   request_info = Request(data)
   request_info.save()
+
+  # # Finds/sends emails. Needs update to use link for the request page. 
+  emails = [contributor.email for contributor in campaign.contributors]
+  if len(emails) > 0: 
+    request_notification(emails, 'Test Link')
```

Comment:
Eventually, I'd like to pass in the page where a contributor can view a request to this form. For now its just a dummy link.